Documentation/writing-principles-and-guidelines/spelling-and-typo3-terminology.rst

```diff
@@ -1,111 +1,1006 @@
+
+        Content blocks are predefined layouts and content elements that can be
+        used to create page content in the TYPO3 backend. Currently it refers to
+        an extension, whose feature set shall be part of TYPO3 v13. Content
+        Blocks are small sets of configuration, defining both Backend input and
+        frontend output.
+
+    ..  _content-elements:
+    -   *Content Elements*
+
+        Content elements in TYPO3 are building blocks of content that can shown
+        in the frontend. Each content element has many (and also custom)
+        attributes, even consisting of nested hierachies of further content
+        elements.
+
+    ..  _core:
+    -   *Core*
+
+        The TYPO3 Core is the central framework of the CMS that provides all
+        basic functions and features.
+
+    ..  _core-development:
+    -   *Core Development*
+
+        Core Development refers to the development and maintenance of the
+        central TYPO3 framework by the Core Team.
+
+    ..  _core-merger:
+    -   *Core Merger*
+
+        A Core Merger is a person or team member responsible for merging code
+        changes and updates into the TYPO3 core. TYPO3 Core Mergers are elected
+        in a formal process.
+
+    ..  _core-team:
+    -   *Core Team*
+
+        The Core Team consists of the main developers and contributors
+        responsible for developing and maintaining the TYPO3 core.
+
+    ..  _crop-variants:
+    -   *Crop variants*
+
+        Crop Variants are different cropping options for images that can be
+        defined and used within the TYPO3 system. One image can have for example
+        a crop variant for "mobile" and "desktop", or different aspect ratios,
+        or any custom scheme.
+
+    ..  _crowdin:
+    -   *Crowdin*
+
+        Crowdin is a translation tool used for localizing and translating TYPO3
+        content into different languages.
+
+    ..  _ctype:
+    -   *CType*
+
+        :sql:`CType` stands for Content Type and is a database column field of
+        the vital "tt_content" database table, where all content elements are
+        stored. This column defines the name of a specific element, and
+        influences how it is presented in the backend and frontend.
+
+*   D
+
+    ..  _dashboard---widgets:
+    -   *Dashboard / Widgets*
+
+        The dashboard is a customizable start page in the TYPO3 backend that
+        contains various widgets for displaying important information and quick
+        access.
+
+    ..  _data-processor:
+    -   *Data Processor*
+
+        A data processor is a component that processes and manipulates data
+        before it is displayed in the TYPO3 frontend. Data processors are
+        implemented as PHP code and can be executed through TypoScript
+        configuration, and affect data passed on to Fluid templates. Thus, it is
+        a specific implementation to operate on data before it is passed on to
+        the presentation layer (templates).
```

Comment:
Data Processors are an integral part of Fluid. I think we can and should name "Fluid" here